pm/validator.go

```diff
@@ -31,12 +32,12 @@ type Validator interface {
 
 // validator is an implementation of the Validator interface
 type validator struct {
-	sigVerifier   SigVerifier
+	sigVerifier   lpcrypto.SigVerifier
```

Comment:
I think we should keep the `SigVerifier` interface in the `pm` package so that the `pm` package defines the interface it requires for signature verification as opposed to relying on an interface described by a different package. The implementation provided by the `crypto` package can just adhere to this interface.

Reply:
done in c813aa883b728eafcb616f266566e29505c8d832